packages/x-components/src/x-modules/experience-controls/components/experience-controls.vue

```diff
@@ -0,0 +1,53 @@
+<script lang="ts">
+  import Vue from 'vue';
+  import { Component } from 'vue-property-decorator';
+  import { XOn } from '../../../components/decorators/bus.decorators';
+  import { XEvent, XEventsTypes } from '../../../wiring/events.types';
+
+  /**
+   *This component subscribes to changes in the ExperienceControls module to fire the events that propagate the configuration.
+   *
+   * @param events - The list of the events on the module.
+   */
+  import { xComponentMixin } from '../../../components/x-component.mixin';
+  import { experienceControlsXModule } from '../x-module';
+  @Component({
+    mixins: [xComponentMixin(experienceControlsXModule)]
+  })
+  export default class ExperienceControls extends Vue {
+    /*
+     * Iterates the list of XEvents received and emits them
+     */
```

Comment:
This is just a comment, not a documentation block. Document the function properly.